app.py

```diff
@@ -20,33 +20,84 @@
 
 import sys
 
-from pathlib import Path
 import streamlit as st
 
 from src.captcha_ import captcha_control
 from src.common import page_setup, save_params
 
+from st_pages import Page, show_pages
+
 params = page_setup(page="main")
 
 
+
+def flashdeconvPages():
+    show_pages([
+        Page("app.py", "FLASHViewer", "🏠"),
+        Page("pages/FLASHDeconvWorkflow.py", "Workflow", "⚙️"),
+        Page("pages/FLASHDeconvDownload.py", "Download", "⬇️"),
+        #Page("pages/FileUpload.py", "File Upload", "📁"),
```

Comment:
why commented?

Reply:
File Upload is now handled in the Workflow section. There will be some adjustments necessary regarding the file paths. This was a quick implementation from my side to circumvent the issues with building `FLASHDeconv` for windows when contrib was not available.

Reply:
okay I'll try to make it change in some way... I thought it was weird that even LayoutManager and Viewer sections are all commented out